src/js/app.js

```diff
@@ -14,36 +14,41 @@ const app = new Vue({
     input: '',
 
     include_similar: false,
-    sort_options: SORT_OPTIONS,
+    sort_options: SORT_OPTIONS_SHORT,
     sort: 'stroke',
 
     candidates: [],
     radical_selection: undefined,
 
     searching: false,
+
+    isDropdownOpen: false,
+    isIncludeSimilar: false,
   },
   template: `
     <div class="app">
-      <div>
-        <input type="text" v-model="input" v-on:change="lookup">
+      <div class="search">
+          <div class="search_sort" v-on:click="openDropdown" v-bind:class="{ open: isDropdownOpen}">
+            <div class="search_sortTrigger"><span>{{ sort_options[sort] }}</span>
+            </div>
+            <div class="search_sortOptions">
+              <sort-option v-for="(label, order) in sort_options"
+                                v-bind:key="order"
```

Comment:
I know Maddy doesn't like this, but would you mind if we line up the properties?

```html
<sort-option v-for="(label, order) in sort_options"
                  v-bind:key="order"
                  v-bind:order="order"
                  ...
```
　↓
```html
<sort-option v-for="(label, order) in sort_options"
             v-bind:key="order"
             v-bind:order="order"
             ...
```